modules/api/src/main/java/org/apache/ignite/lang/TableNotExistsException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.lang;
+
+/**
+ * This exception is thrown when a table failed to be dropped or altered, because there is no table with the name.
+ */
+public class TableNotExistsException extends IgniteException {
+    /**
+     * Create a new exception with given table name.
+     *
+     * @param name Table name.
+     */
+    public TableNotExistsException(String name) {
+        super(LoggerMessageHelper.format("Table does not exist [name={}]", name));
```

Comment:
I suppose we need to unify all such exceptions, lets add "name=" all neighbors i.e. ColumnNotFoundException, IndexNotFoundException, or (i vote for this) remove "name=" from all of them. 
```suggestion
        super(LoggerMessageHelper.format("Table '{}' does not exist.", name));
```

Reply:
I do not see a reason to change the message, because we already used the similar format for TableAlreadyExistsException.